_posts/2015-09-01-visibility-is-opportunity.markdown

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title:  "Visibility Is Opportunity: stories of improving continuous integration"
+status: public
+type: post
+author: Xiao Li
+excerpt: <p>Mingle is a 9 years JRuby Rails project. We do test-driven development and continous integration. We write selenium tests to test everything since Oct 2006. For a long time, our build time was 1 to 2 hours. We thought it was a good balance of build time and resources needed...</p>
+---
+
+Visibility can be hard, but it's worth it.
+==================
+
+[Mingle] is a 9 years JRuby Rails project.
```

Comment:
Not sure what you mean.

Sent from my iPhone

> On Sep 1, 2015, at 8:16 PM, Ketan Padegaonkar <notifications@github.com> wrote:
> 
> In _posts/2015-09-01-visibility-is-opportunity.markdown:
> 
> > @@ -0,0 +1,192 @@
> > +---
> > +layout: post
> > +title:  "Visibility Is Opportunity: stories of improving continuous integration"
> > +status: public
> > +type: post
> > +author: Xiao Li
> > +excerpt: <p>Mingle is a 9 years JRuby Rails project. We do test-driven development and continous integration. We write selenium tests to test everything since Oct 2006. For a long time, our build time was 1 to 2 hours. We thought it was a good balance of build time and resources needed...</p>
> > +---
> > +
> > +Visibility can be hard, but it's worth it.
> > +==================
> > +
> > +[Mingle] is a 9 years JRuby Rails project.
> Did you mean to link to the mingle site here?
> 
> —
> Reply to this email directly or view it on GitHub.
>